### ISSUE TITLE
cannot locate jsk_recognition_utils python nodes

### ISSUE THREAD

**Issue by knorth55** [jsk-ros-pkg/jsk_recognition#2117]
https://travis-ci.org/start-jsk/jsk_apc/builds/243660345?utm_source=github_status&utm_medium=notification
```
]]></system-out>
  <system-err><![CDATA[?[31mERROR: cannot launch node of type [jsk_recognition_utils/bounding_box_array_publisher.py]: can't locate node [bounding_box_array_publisher.py] in package [jsk_recognition_utils]?[0m
```

**Comment by knorth55** [comment]
This error occurs when I use debian package and does not occur with source.

**Comment by knorth55** [comment]
Waiting for new release of jsk_recognition_utils.

**Comment by knorth55** [comment]
I checked this issue is solved in 1.1.3
